### PR TITLE
Add cod.m zigbee coordinator to zha docs

### PR DESCRIPTION
**Pull request by tks09**
## Proposed change

Add cod.m zigbee coordinator to zha docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/115471
- Link to parent pull request in the Brands repository: -
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Introduced support for a new Zigbee Coordinator device, `cod.m Zigbee Coordinator`, with Zeroconf functionality under the service name `czc._tcp.local`.

### CODE REVIEW

**Comment by tks09** [comment]
Not really sure why the netlify did deploy did not work when resolving dependencies, since there were no changes. If I did something wrong I would appreciate a pointer 🙇‍♂️

**Comment by puddly** [comment]
The URL 404s: https://docs.codm.de/zigbee/zigbee-coordinator/

**Comment by tks09** [comment]
Thanks for checking, we will to provide the URL in the next days on our docs system, should the PR be put on wait?

**Comment by tks09** [comment]
We released the documentation page. Is there any way I can trigger a rerun of the PR checks?

**Comment by c0ffeeca7** [comment]
> We released the documentation page. Is there any way I can trigger a rerun of the PR checks?

We are seeing this issue also in some other PRs. Please be patient. Someone with more wizardry-magic powers than me will take a look at this.

**Comment by tks09** [comment]
> Please be patient. 

Okay cool, no problem for us! I was just not sure if did anything wrong on our side which I can change to fix this. Thanks for the reply!

**Comment by frenck** [comment]
@tsmt09 Can you please rebase the PR onto the upstream (our) `next` branch?

**Comment by tks09** [comment]
Thanks for the hint, I did the rebase and it seems to run now!

**Comment by tks09** [comment]
rebased to next

**Comment by tks09** [comment]
rebased to next

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -214,6 +214,7 @@ Some devices can be auto-discovered, which can simplify the ZHA setup process. T
 | [ZigStar Coordinators](https://zig-star.com/)                                                                                               | USB              | 1A86:7523                      |
 | [SMLIGHT SLZB-06 POE Zigbee LAN WiFi USB Adapter](https://smlight.tech/product/slzb-06/)                                                    | Zeroconf         | slzb-06.local.                 |
 | [ZigStar UZG Universal Zigbee Gateway (UZG-01)](https://uzg.zig-star.com)                                                                   | Zeroconf         | uzg-01._tcp.local.             |
+| [cod.m Zigbee Coordinator](https://docs.codm.de/zigbee/zigbee-coordinator/)                                                                 | Zeroconf         | czc._tcp.local.                |
```
Comment (TheJulianJES) [inline source/_integrations/zha.markdown]: This URL redirects to https://docs.codm.de/zigbee/coordinator/ now.
Should this be updated or is this intentional and should stay as-is?

Comment (tks09) [reply source/_integrations/zha.markdown]: Thanks for the hint, changed

**Comment by tks09** [comment]
rebased

**Review by TheJulianJES — APPROVED** [review]

**Pull request merged**